react-native-purchases-ui/ios/RNPaywalls.m

```diff
@@ -58,24 +58,49 @@ - (PaywallProxy *)paywalls API_AVAILABLE(ios(15.0)){
 
 // MARK: -
 
-RCT_REMAP_METHOD(presentPaywall,
-                 presentPaywallWithResolve:(RCTPromiseResolveBlock)resolve
-                 reject:(RCTPromiseRejectBlock)reject) {
+RCT_EXPORT_METHOD(presentPaywall:(NSDictionary *)options
+                  withResolve:(RCTPromiseResolveBlock)resolve
+                  reject:(RCTPromiseRejectBlock)reject) {
     if (@available(iOS 15.0, *)) {
-        [self.paywalls presentPaywallWithPaywallResultHandler:^(NSString *result) {
+        NSString *offeringIdentifier = options[@"offeringIdentifier"];
+        NSNumber *displayCloseButton = options[@"displayCloseButton"];
+
+        if (displayCloseButton == nil) {
+            [self.paywalls presentPaywallWithOfferingIdentifier:offeringIdentifier
+                                           paywallResultHandler:^(NSString *result) {
+                resolve(result);
+            }];
+            return;
+        }
+        [self.paywalls presentPaywallWithOfferingIdentifier:offeringIdentifier
```

Comment:
So we are using offering id in iOS. It's only in android that needs to be fixed, correct?

Reply:
ugh this is also not intended, I have some local work on purchases hybrid common, but this should point to the current version of phc, I forgot to revert it